app/client/src/styles/components/home.styl

```diff
@@ -32,7 +32,7 @@
       font-family Earth Orbiter Title, Open Sans, sans-serif
       font-size 1.75em
       font-weight 600
-      text-transform uppercase
+      text-transform none
```

Comment:
Instead of using `text-transform none`, I think it would be better just to remove the line entirely as it isn't really a needed style.

Reply:
I get your point but the problem persists if I just delete the line. It seems to inherit the attribute from the <h1> tag. ![image](https://user-images.githubusercontent.com/26444315/30246629-2f2fdcd8-9608-11e7-96f4-27f365cb8451.png)

Reply:
Ahh, nice catch. This issue is coming from here:

https://github.com/saadq/latexresu.me/blob/6f11b269b67214d3417c44d475d176b4be8849a4/app/client/src/styles/components/templates.styl#L4-L9

Those styles are meant to be used for each section heading in the resume generator. For some reason, I accidentally made them global and put them in `templates.styl` for whatever reason. 

So if you'd like to fix this as well, there are probably 2 steps:

1. Move that `h1` code to be a nested child of the `.form-section`:

https://github.com/saadq/latexresu.me/blob/6f11b269b67214d3417c44d475d176b4be8849a4/app/client/src/styles/components/content.styl#L40-L44

2. Remove the `letter-spacing` and `font-size` code from there and move it to the `h1` styles in `home.styl`.

Let me know if you'd like any clarification! :)

Reply:
You're right, everything seems to be ok now